package.json

```
@@ -133,7 +133,7 @@
         "rimraf": "^3.0.2",
         "sandboxed-module": "^2.0.4",
         "ts-node": "^9.0.0",
-        "typescript": "^3.9.7",
+        "typescript": "4.1",
```

Just wondering, why not to update to the latest typescript version?

```suggestion
        "typescript": "4.9.3",
```

---

One of the reasons was the latest typescript version increases our browser build size by a substantial amount, as well i dont believe theres a reason to upgrade as these changes wont affect us. we only use typescript for the build process on our own type declaration files.